### PR TITLE
[DO NOT MERGE] Jaw/v0.18.0

### PR DESCRIPTION
**Pull request by jwoertink**
This is basically #413 but branched from c45e1860751bba25a16120402f93e7537c0be5b5

### CODE REVIEW

File: src/web_app_skeleton/.crystal-version [hunk 1]
```diff
@@ -1 +1 @@
-0.30.0
+0.30.1
```
Comment (paulcsmith) [inline src/web_app_skeleton/.crystal-version]: I  think this is supposed toe 0.31.0

File: src/lucky_cli/version.cr [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module LuckyCli
-  VERSION = "0.17.0-rc1"
+  VERSION = "0.17.0"
```
Comment (paulcsmith) [inline src/lucky_cli/version.cr]: Should this  be 0.18.0?

File: src/web_app_skeleton/.crystal-version (hunk 1 above)
Comment (jwoertink) [reply src/web_app_skeleton/.crystal-version]: lol yup. 😅

File: src/lucky_cli/version.cr (hunk 2 above)
Comment (jwoertink) [reply src/lucky_cli/version.cr]: ugh 😫 yeah. I knew I'd miss stuff.

**Comment by paulcsmith** [comment]
I  think we should also sneak this into this release since it fixes the issue where it verifies the connection before the db is created https://github.com/luckyframework/lucky_cli/commit/aa2cb7a4946e59cb07b83fadd773d0b2cfaa7003#diff-154806d6e0faa0b1aa1e518f3bbd3647

**Comment by paulcsmith** [comment]
We've decided to make 0.18 The Big One™  so we'll close this for now

**Pull request closed without merging**